ifStatement.js

```diff
@@ -33,6 +37,11 @@ Steps:
     - If either of the conditions is false, print the message Hello ${enteredUsername}, I'm sorry but it seems you're not authorized to access the restricted area.
 ***********************************************************/
 // TODO: ADD YOUR CODE BELOW
+let userName="ameer"
+let role="admin"
+let username= prompt ('enter your name')
+if (userName==username&&role=="admin"){console.log(` Hello ${userName} , and you have permission to access the restricted area.`);}
```

Comment:
1. It's not recommended to use the same variable name with different cases like `username` and `userName` as they refer to different values. 
You can differentiate between which refers to which: use clear, concise names like `username` and `usernameInput` or `enteredUsername`.

2. Also, for the block of code within the curly brackets of the if statement, please write the block of code on the next line, not right next to the curly bracket. 
This makes your code more organized and readable.